realm/realm-library/src/main/java/io/realm/internal/Collection.java

```diff
@@ -594,6 +711,16 @@ private static native long nativeCreateResultsFromLinkView(long sharedRealmNativ
 
     private static native void nativeDelete(long nativePtr, long index);
 
+    private static native void nativeSetNull(long nativePtr, long columnIndex);
+    private static native void nativeSetBoolean(long nativePtr, long columnIndex, boolean value);
+    private static native void nativeSetInt(long nativePtr, long columnIndex, long value);
```

Comment:
mmmm.... We should revisit once this is in SO and has proper signatures.
I believe, though, that core refers to the type used to represent byte, short, int and long, as Int.  I want to use the name of the core type, since I will call it from the `set` methods for all four java types.